docs/2-how-it-works/momento-concepts.md

```diff
@@ -157,15 +157,9 @@ To set a custom TTL value in a set operation using Momento's Python SDK, use cod
 cache_client.set("users-cache", "foo", "123", 100)
 ```
 
-In this operation, we are setting the cache entry identified by the key of "foo" to the value of "123" within the "users-cache" in our Momento account. For this important value, we've decided it should be cached for 100 seconds rather than the default of 60 seconds.
+In this operation, we are setting the cache entry identified by the key of "foo" to the value of "123" within the "users-cache" in our Momento account. For this important value, we've decided it should be cached for 100 seconds rather than the default of 60 seconds you specified when initializing your client.
```

Comment:
```suggestion
In this operation, we are setting the cache entry identified by the key of "foo" to the value of "123" within the "users-cache" in our Momento account. For this important value, we've decided it should be cached for 100 seconds rather than the default of 60 seconds we specified when initializing our client.
```